extensions/jupyter-adapter/src/JupyterKernel.ts

```diff
@@ -726,7 +726,12 @@ export class JupyterKernel extends EventEmitter implements vscode.Disposable {
 		const command = args.join(' ');
 
 		// Create environment.
-		const env = { POSITRON: '1', POSITRON_VERSION: positron.version };
+		const env = {
+			POSITRON: '1',
+			POSITRON_VERSION: positron.version,
+			POSITRON_LONG_VERSION: `${positron.version}+${positron.buildNumber}`,
+			POSITRON_UI_KIND: vscode.env.uiKind === vscode.UIKind.Desktop ? 'desktop' : 'web',
+		};
```

Comment:
This adds these new env vars to the Python and R consoles. Do we also need these in the terminal?

Reply:
In the RStudio terminal, I see `RSTUDIO_PROGRAM_MODE=desktop` but no version info.

In the VS Code terminal, I see `TERM_PROGRAM_VERSION=1.93.1` but no info about web vs. desktop.

Reply:
They're POSITRON_ prefixed, so seems fine to include version and mode information, if helpful. Don't love the terminology UI_KIND though, would have preferred something like CLIENT_MODE, though it looks like that is a vscode env term.